jdbc/core/src/main/java/org/apache/shardingsphere/driver/jdbc/core/driver/ShardingSphereDriverURL.java

```diff
@@ -72,4 +72,22 @@ public byte[] toConfigurationBytes() {
             return builder.toString().getBytes(StandardCharsets.UTF_8);
         }
     }
+    
+    private InputStream getResourceAsStream(final String resource) throws IOException {
+        ClassLoader[] classLoaders = new ClassLoader[]{
+                Thread.currentThread().getContextClassLoader(), getClass().getClassLoader(), ClassLoader.getSystemClassLoader(),
+        };
+        for (ClassLoader each : classLoaders) {
+            if (null != each) {
+                InputStream result = each.getResourceAsStream(resource);
+                if (null == result) {
+                    result = each.getResourceAsStream("/" + resource);
+                }
```

Comment:
Could we bind resource path with class loader? Maybe it's more efficient and readable.
e.g. `TCCL` just use `resource`, `getClass().getClassLoader()` just use `"/" + resource`

Reply:
1, Could we remove the last `,` after `ClassLoader.getSystemClassLoader()`?

2, And did you try to run `mvn spotless:apply` in this module? Seems it might change the code formatting.

Reply:
I added  `,` in the arrays due to the checkstyle failures
```
[ERROR] src/main/java/org/apache/shardingsphere/driver/jdbc/core/driver/ShardingSphereDriverURL.java:[79] (coding) ArrayTrailingComma: 数组尾元素后应有逗号','。
```
So it need to change the `checkstyle.xml` to remove `<module name="ArrayTrailingComma" />` ?

yeah, I run `mvn splotless:apply` and it looks good.

Reply:
And I think we can not bind `/` with class loader.  See my above explaination.

Reply:
IIRC all `ClassLoader` uses relative path to load the resources. But in some case of custom classloader, it might need the leading of `/`. So it makes sense to try `resource` at first and if it fails, try `"/" + resource`.